ntbs.sln

```diff
@@ -15,7 +15,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ntbs-ui-tests", "ntbs-ui-te
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EFAuditer-tests", "EFAuditer-tests\EFAuditer-tests.csproj", "{245743CB-8827-44E6-8AA2-361E1074B2EE}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "coming-soon", "coming-soon\coming-soon.csproj", "{ADA957B5-4B6E-44AD-8AC1-9346BC07DD0B}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "temporarily-unavailable", "temporarily-unavailable\temporarily-unavailable.csproj", "{ADA957B5-4B6E-44AD-8AC1-9346BC07DD0B}"
```

Comment:
I had some more thoughts about this, and I think the right thing to do might be to move this project to its own solution. It's essentially an entirely independent project, so I don't think there's any benefit to lumping them together. What do you think?